client/state/posts/test/reducer.js

```diff
@@ -205,7 +238,7 @@ describe( 'reducer', () => {
 			} );
 		} );
 
-		it( 'never persists state because this is not implemented', () => {
+		it( 'never persists state', () => {
```

Comment:
We want to persist queries in the future, but not the other temporary fetch state, so posts/siteRequests and posts/queriesLastPage should not be persisted, while posts/queries will be in the future.